pkg/controller/user/update.go

```diff
@@ -103,6 +103,12 @@ func (c *Controller) HandleUpdate() http.Handler {
 	})
 }
 
+func (c *Controller) renderUpdate(ctx context.Context, w http.ResponseWriter, user *database.User) {
+	m := controller.TemplateMapFromContext(ctx)
+	m["user"] = user
+	c.h.RenderHTML(w, "users/new", m)
```

Comment:
It's a little weird that `renderUpdate` is named "new"

Reply:
new and update share the same page...